extensions/emmet/src/abbreviationActions.ts

```diff
@@ -264,47 +264,47 @@ export async function wrapWithAbbreviation(args: any): Promise<boolean> {
 }
 
 export function expandEmmetAbbreviation(args: any): Thenable<boolean | undefined> {
-	if (!validate() || !vscode.window.activeTextEditor) {
-		return fallbackTab();
+	if (!validate()) {
+		return Promise.resolve(undefined);
 	}
 
+	const editor = vscode.window.activeTextEditor!;
```

Comment:
Are we guaranteed an active text editor here?

Reply:
Yep, otherwise `validate()` above returns `false` and we exit out of the function early.